frontstage/controllers/party_controller.py

```diff
@@ -438,21 +430,25 @@ def get_survey_list_details_for_party(respondent: dict, tag: str, business_party
 
     # This is a dictionary that will store all the data that is going to be cached instead of making multiple calls
     # inside the for loop for get_respondent_enrolments.
-    cache_data = {"collexes": dict(), "cases": dict()}
+    cache_data = {"cases": dict()}
     redis_cache = RedisCache()
 
     # Populate the cache with all non-instrument data
     caching_data_for_survey_list(cache_data, surveys_ids, business_ids, tag)
 
-    enrolments = get_respondent_enrolments_for_started_collex(enrolment_data, cache_data["collexes"])
+    #  Populate the enrolments by creating a dictionary using the redis_cache
+    enrolments = get_respondent_enrolments_for_started_collex(
+        enrolment_data, dict((s_id, redis_cache.get_collection_exercise(s_id)) for s_id in surveys_ids)
```

Comment:
Change s_id to `survey_id`.  Easier to read at a glance and letters aren't at a premium ;)